docs/chat.md

```diff
@@ -268,6 +292,7 @@ See [OCP\RichObjectStrings\Definitions](https://github.com/nextcloud/server/blob
 * `guest_moderator_promoted` - {actor} promoted {user} to moderator
 * `guest_moderator_demoted` - {actor} demoted {user} from moderator
 * `message_deleted` - Message deleted by {actor} (Should not be shown to the user)
+* `cleared_history` - {actor} cleared the history of the conversation
```

Comment:
this key isn't using the same pattern like the others

all the others are "$object_$verb" but here you used "$verb_$object".

</picky_mode>

Reply:
makes sense